arlas-core/src/main/java/io/arlas/server/services/ExploreServices.java

```diff
@@ -483,6 +525,13 @@ public AggregationResponse formatAggregationResult(MultiBucketsAggregation aggre
         return keyStringifier.apply(flatted);
     }
 
+    private boolean isAggregatedGeometry(String subName, List<AggregatedGeometryEnum> geometries) {
+        if(!CollectionUtils.isEmpty(geometries)) {
+            long n = geometries.stream().map(g -> g.value() + FluidSearch.AGGREGATED_GEOMETRY_SUFFIX).filter(g -> g.equals(subName)).count();
```

Comment:
there is a anyMatch method on the stream, that looks to fit your need (instead of the `count`)

Reply:
fixed